src/Context/BasicContext.php

```diff
@@ -1295,17 +1295,21 @@ protected function logException(Exception $exception)
      * There's already an xpath based function 'I see the "" element' iSeeTheElement() in silverstripe/cms
      * There's also an 'I should see "" element' in MinkContext which also converts the css selector to xpath
      *
-     * @When /^I should see the "([^"]+)" element/
+     * @When /^I should(| not) see the "([^"]+)" element/
      * @param $selector
      */
-    public function iShouldSeeTheElement($selector)
+    public function iShouldSeeTheElement($not, $selector)
```

Comment:
That's changing an API. Even if most people will only interact with this method via their feature file, there's a possibility that someone could call it directly from their custom context.

Reply:
> there's a possibility that someone could call it directly 

:laughing: 

ok fine, easier to just respect semver than arguing about it.  I'll make the function backwards compatibile